apps/OpenSpace/main.cpp

```diff
@@ -574,60 +583,61 @@ void mainPostDrawFunc() {
 
 
 void mainKeyboardCallback(sgct::Key key, sgct::Modifier modifiers, sgct::Action action,
-                          int)
+                          int, sgct::Window* window)
```

Comment:
This argument would be on a new line since we have overflown the column limit one

Reply:
No? Not if I interpret our coding style document correctly. We align the arguments on the new lines wiht the `(` bracket and only add new lines when we reach the column limit. At least according to the second example here:

![image](https://user-images.githubusercontent.com/8808894/182549696-a002d984-e554-4a90-873a-a402f481c8ba.png)

Reply:
Oops... yes, you are right!  I think I'll have to fix some of my misindentations then!